src/test/java/io/confluent/connect/hdfs/avro/HiveIntegrationAvroTest.java

```diff
@@ -244,21 +258,13 @@ public void testHiveIntegrationFieldPartitionerAvro() throws Exception {
 
     assertEquals(expectedColumnNames, actualColumnNames);
 
-    List<String> partitionFieldNames = connectorConfig.getList(
-        PartitionerConfig.PARTITION_FIELD_NAME_CONFIG
+    List<String> expectedPartitions = Arrays.asList(
+            partitionLocation(TOPIC, 16, "int"),
```

Comment:
should we extract "int" as a variable called partitionFieldName as I had to read some code to figure this out:)?

Reply:
Any decent IDE would inline the name of the argument :)
![image](https://user-images.githubusercontent.com/215270/124563382-d7f74f80-ddf4-11eb-9654-fdc5e31d9e98.png)